app/scripts/controllers/remark.ctrl.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * (c) 2016 Tieto Finland Oy
+ * Licensed under the MIT license.
+ */
+'use strict';
+
+/**
+* @ngdoc function
+* @name dashboard.controller:remarkCtrl
+* @description
+* # remarkCtrl
+* Controller of the dashboard
+*/
+angular.module('dashboard')
+    .controller('remarkCtrl', ['$log', '$scope', 'StorageSrv', 'CONST', function ($log, $scope, StorageSrv, CONST) {
+        $log.debug("remarkCtrl: CONTROLLER");
+        var self = this;
+        self.topic = StorageSrv.getKey(CONST.KEY.TOPIC);
+
+        if (!angular.isObject(self.topic)) {
```

Comment:
Should this check be before assigning, and skip if it's not valid? Then template can assume property is etiher a `{}` or a valid object? Just a thought.